File: src/main/webapp/resource/user/selenium/js/me.js
/**
 * author:selenium
 */
var APP_PATH = $("#APP_PATH").val();
$(function() {
	selectMe();// 查询个人信息
	$("#submitButton").click(function(){
		addContact();
		return false;//阻止表单的默认提交
	});
});
function selectMe() {// 查询个人信息
	$.ajax({
		url : APP_PATH + "/user/selectMe",
		type : "GET",
		success : function(result) {
			var me = result.data.me;
			$("#name").append(me.name);
			$("#nickname").append(me.nickname);
			me.gender == 0 ? $("#gender").append("女") : $("#gender")
					.append("男");
			$("#age").append(me.age);
			$("#phone").append(me.phone);
			$("#email").append(me.email);
			$("#title").append(me.title);
			$("#viceTitle").append(me.viceTitle);
			$("#introduction").append(me.introduction);
		}
	});
}

function addContact(){
	$.ajax({
		url:APP_PATH+"/user/addContact",
		type:"post",
		data:$("#contactForm").serialize(),
		success:function(result){
			 alert("I have received your letter and I will contact you.");
			 window.location.reload();
		}
	});
}

File: src/main/java/com/wantao/bean/Me.java
package com.wantao.bean;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * @author wantao
 * @date 2019-02-27 11:27
 * @description: 个人信息类
 */

/**
 * @param id 个人id
 * @param name 个人姓名
 * @param avatar 个人头像
 * @param password 个人密码
 * @param nickname 个人昵称
 * @param gender 个人性别 1:男性 2:女性
 * @param age  个人年龄
 * @param phone 个人电话
 * @param email 个人邮箱
 * @param title 个人简介标题
 * @param viceTitle 个人简介副标题
 * @param introduction 个人简介内容
 */
@Data
@AllArgsConstructor
public class Me implements Serializable{
	private static final long serialVersionUID = 8074496952343192128L;
	private Integer id;
	private String name;
	private String avatar;
	private String password;
	private String nickname;
	private Integer gender;
	private Integer age; 
	private String phone;
	private String email;
	private String title;
	private String viceTitle;
	private String introduction;
	public Me() {};

}
